docs/DeveloperGuide.md

```diff
@@ -408,6 +418,33 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     * 2b1. System gives corresponding error message.
 
       Use case ends.
+      
+**UC08 - User clears client list**
+
+**MSS**
+
+1. User chooses to clear all client list.
```

Comment:
Perhaps we can say ...clear the entire...

Reply:
i think ill also include client list AND archive list

Reply:
Ps then we should say "clear the active client list and archived client list."